schema_based_on_diagram.sql

```diff
@@ -0,0 +1,47 @@
+CREATE DATABASE vet_clinic_diagram;
+-- Create a table named invoice_items  with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+--unit_price        DECIMAL,
+ -- quantity          INT,
+  --total_price       DECIMAL,
+  --invoice_id        INT REFERENCES invoices(id),
+  --treatment_id      INT REFERENCES treatments(id),
+  --PRIMARY KEY(id, invoice_id, treatment_id)
```

Comment:
- [ ] Please create a "join table" for the many-to-many relationships between medical_histories and treatments as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/231500478-244ecf95-8dbb-43f2-a426-9c0545d1905a.png)

Reply:
Thanks for sharing th project requirements again. I wil lcreate the new join table